core/src/main/java/fr/sncf/osrd/standalone_sim/StandaloneSim.java

```diff
@@ -172,7 +172,7 @@ public static Optional<AbstractAllowanceWithRanges> generateAllowanceFromSchedul
         double lostTime = 0.;
         for (var schedulePoint : scheduledPoints) {
             var rangeEndPos = schedulePoint.pathOffset;
-            double excessTime = schedulePoint.time - maxEffortEnvelope.interpolateTotalTime(rangeEndPos) - lostTime;
+            double excessTime = schedulePoint.time - maxEffortEnvelope.interpolateDepartureFrom(rangeEndPos) - lostTime;
```

Comment:
```suggestion
            double excessTime = schedulePoint.time - maxEffortEnvelope.interpolateArrivalAt(rangeEndPos) - lostTime;
```

It doesn't match the previous version, but it was not working properly

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/7609/commits/b7f044ebd08664f02948f7aafb19982fee469f65

As discussed, we should add integration tests for v2 on that (issue to be opened):
Start by a 30 min stop, roll for 10 min, then stop again for 10, then finish in 10 min.
Add a general allowance.
Check that we have correct times (stops not impacted by allowance) and correct warnings if necessary and implemented.
Maybe trigger internal asserts on the way.